kubernetes/process_manifest.rb

```diff
@@ -3,6 +3,7 @@
 require "util"
 require "yaml"
 require "fileutils"
+require 'json'
```

Comment:
SUSE has its own conventions when it comes to ruby hacking. One of them is the double quotes-always rule.

However, and as a side note: there's no performance benefit in single quote nowadays.